src/render-ui.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { mount } from '@cypress/react';
+import Chatbox from './components/Chatbox';
+import { CHAT_ID, CHAT_MESSAGES } from '../cypress/fixtures/chat_messages';
+import { ImmutableMember, Member } from './types';
+import { MEMBERS } from '../cypress/fixtures/members';
+import { List } from 'immutable';
+import { mockUseAvatar } from '../cypress/fixtures/mockHooks';
+
+describe('Render Avatar', () => {
+  beforeEach(() => {
+    mount(
+      <Chatbox
+        chatId={CHAT_ID}
+        currentMember={new ImmutableMember(MEMBERS.ANNA)}
+        members={List(Object.values(MEMBERS) as Member[])}
+        messages={List(CHAT_MESSAGES)}
+      />,
+    );
+  });
+
+  it('should call avatar hook', () => {
+    const mockUseAvatarHookName = 'useAvatarHook';
+    const fakeHook = mockUseAvatar(mockUseAvatarHookName);
```

Comment:
Nice! I guess its just a preference, but you could defined the name in the function, and the function could return the name. Or the name could also be a constant.